### PR TITLE
Feat/insufficient gas

### PR DESCRIPTION
**Pull request by valpinkman**
Check for gas when sending ERC20 tokens

depends on #1002 

### Type

Feat

### Context

LL-1621

### Parts of the app affected / Test plan

Send Flow

IMG_C2CDB9B31163-1

### CODE REVIEW

File: src/locales/en/common.json [hunk 1]
```diff
@@ -205,6 +205,9 @@
       "title": "Oops, insufficient balance",
       "description": "Make sure the account to debit has sufficient balance"
     },
+    "NotEnoughGas": {
+      "title": "Oops, insufficient gas"
```
Comment (juan-cortes) [inline src/locales/en/common.json]: >_ETH balance in parent account insufficient for network fees_

According to https://github.com/LedgerHQ/ledger-live-desktop/blob/63b38a673c9d3d50b7997014557f37888b49dc7f/static/i18n/es/app.json#L1056

Comment (juan-cortes) [reply src/locales/en/common.json]: 🤷‍♂

Comment (gre) [reply src/locales/en/common.json]: Yeah agree we need same wording. If we can change the wording, please also update the screenshot before we confirm with the team this is where we need the fix to be.

**Comment by valpinkman** [comment]
IMG_4C8B4E995185-1

the text seems WWAAAAYYYY too long for me.

@gre @dekkaki @khalilbenihoud

**Comment by juan-cortes** [comment]
That looks disgustingly long  👀

**Comment by gre** [comment]
I think the point is more to start a discussion anyway, we're not sure what is the best solution yet but it's important we at least want to block at this point because you shouldn't be able to go farther than this if you can't spend the transaction right?

we should have an internal discussion what what would be a better UX if we want something better :)

**Comment by gre** [comment]
=> probably need to move the error on the summary so we can change the gas afterwards and also handle case where you would augment the gas.

**Review by faith — APPROVED** [review]

**Comment by LFBarreto** [comment]
the issue seems fixed now closing this

**Pull request closed without merging**